diff --git a/backend/alembic/versions/v290_dashboards_presets.py b/backend/alembic/versions/v290_dashboards_presets.py
index 5ee832a3..75ef1d31 100644
--- a/backend/alembic/versions/v290_dashboards_presets.py
+++ b/backend/alembic/versions/v290_dashboards_presets.py
@@ -74,7 +74,7 @@ def upgrade() -> None:
                 "shared_with_project",
                 sa.Boolean(),
                 nullable=False,
-                server_default=sa.text("0"),
+                server_default=sa.text("false"),
             ),
             sa.Column(
                 "created_at",
